includes/blast_ui.form_advanced_options.inc

```diff
@@ -493,360 +494,8 @@ function blast_ui_blastp_advanced_options_form_submit($form, $form_state) {
 
   // Expand Gap Cost key into open and extend penalties
   $matrix = $form_state['values']['Matrix'];
-  switch ($matrix) {
```

Comment:
Code below moved to getGap() as it will be needed also for web services support.